### PR TITLE
Add source for examples used in docs, fix errors

### PR DESCRIPTION
**Pull request by toerndev**
I noticed a few mistakes in the example code from my last patch. I have added source files and way to run them, to prevent this from happening going forward. Also added the Redux devtools example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +208,20 @@ See the [migration instructions](https://github.com/faceyspacey/redux-first-rout
...
+`yarn add -D redux-devtools-extension`
+```diff
+// configureStore.js
+-import { createStore, applyMiddleware, compose, combineReducers } from 'redux'
++import { createStore, applyMiddleware, combineReducers } from 'redux'
+ import { connectRoutes } from 'redux-first-router'
++import { composeWithDevTools } from 'redux-devtools-extension'
+ 
+-  const enhancers = compose(enhancer, middlewares)
++  const enhancers = composeWithDevTools(enhancer, middlewares)
```
Comment (ScriptedAlchemy) [inline README.md]: there should be come `typeof window !== 'undefined'
```suggestion
+  const toCompose = typeof window !== 'undefined' ? composeWithDevTools : compose
+  const enhancers = toCompose(enhancer, middlewares)
```

Comment (ScriptedAlchemy) [reply README.md]: that way devtools dont leak into prod

Comment (toerndev) [reply README.md]: You're right, I guess I thought the library would handle that under the hood. I'll try to fix this later today.

Comment (ScriptedAlchemy) [reply README.md]: Thanks man :) after that, ill marge

**Comment by toerndev** [comment]
Sorry about the mess with double patches, not sure how to amend without gerrit.
I tried to read a bit about how to use devtools, it seems like you want to check window when working with universal, but then we shouldn't need the npm package anymore. Went with something from some official-looking article, it works but does it look OK to you?

**Comment by toerndev** [comment]
Also changed the example structure a bit, so you can combine them to run the different recipes we'll have in the README.

**Pull request merged**